public/tutorials/robust-smart-contracts-with-openzeppelin.md

```diff
@@ -112,51 +114,102 @@ Using less than 15 lines of hand-coded Solidity, we've created our own Ethereum
+     ... 0xbb0577c172dbb3df061f9b7a87ac22883c6dc62f1423748ebf53e51db735d8b9
+     Migrations: 0x3ed10fd31b3fbb2c262e6ab074dd3c684b8aa06b
+   Saving successful migration to network...
+     ... 0x429a40ee574664a48753a33ea0c103fc78c5ca7750961d567d518ff7a31eefda
+   Saving artifacts...
+   Running migration: 2_deploy_contracts.js
+     Deploying TutorialToken...
+     ... 0x6a37fa398faa57769360cfb7125366ef56895f8885c224312f281d2df3475f08
+     TutorialToken: 0x377bbcae5327695b32a1784e0e13bedc8e078c9c
+   Saving successful migration to network...
+     ... 0x6e25158c01a403d33079db641cb4d46b6245fd2e9196093d9e5984e45d64a866
+   Saving artifacts...
+   ```
 
 ## Interacting with the new token
 
-For this portion of the tutorial, we recommend using the MetaMask extension for Chrome. It will allow you to switch between accounts quickly; perfect for testing the ability to transfer our newly created tokens. Our [Pet Shop tutorial](/tutorials/pet-shop) has more information about [configuring MetaMask with the TestRPC](/tutorials/pet-shop#interacting-with-the-dapp-in-a-browser).
+For this portion of the tutorial, we recommend using the [MetaMask extension for Chrome](http://metamask.io). It will allow you to switch between accounts quickly; perfect for testing the ability to transfer our newly created tokens. Our [Pet Shop tutorial](/tutorials/pet-shop) has more information about [configuring MetaMask](/tutorials/pet-shop#interacting-with-the-dapp-in-a-browser).
 
-1. To run a local web server containing the front-end application, run the following command from the root `oz-workspace` directory:
+1. Leaving Truffle Develop running, open a second terminal in the root of your project directory and run a local web server containing the front-end application:
 
    ```shell
    npm run dev
    ```
 
-A browser window should automatically open with the interface below:
+   A browser window should automatically open with the interface below:
+
+   ![TutorialToken Wallet](/tutorials/images/open-zeppelin/oz-tutorialtoken-initial.png)
+
+   *TutorialToken wallet*
+
+   Our basic dapp shows the TutorialToken balance of the selected account in MetaMask.
+
+1. Now we'll transfer some TutorialToken tokens to a different account. Truffle Develop, when launched, lists 10 accounts. The first account has the token balance. Pick one of the other accounts (we recommend the second account) and enter it in the "Address" box, and also enter `2000` in the "Amount" field.
+
+   ![TutorialToken Wallet transfer recipient](/tutorials/images/open-zeppelin/oz-transfer-address.png)
+
+   *TutorialToken wallet transfer recipient*
+
+   <p class="alert alert-info">
+     <strong>Note</strong>: To review the list of accounts, type `web3.eth.accounts` in Truffle Develop.
+   </p>
+
+1. Click "Transfer" to initiate the token transfer. Metamask will intercept the transfer request and display a confirmation. Note that no ether is changing hands, except in the form of gas.
```

Comment:
I'd probably change the tail end of this sentence to say "except for the gas used to pay for the transaction." Something to suggest the point of the gas.

Reply:
Good point, that's clearer.